omnibus/files/private-chef-cookbooks/private-chef/recipes/elasticsearch.rb

```diff
@@ -159,7 +159,13 @@
 end
 
 component_runit_service 'elasticsearch' do
-  action [:enable, :start]
+  action [:enable]
+end
+
+component_runit_service 'elasticsearch' do
+  action [:start]
+  retries 10
```

Comment:
Might be useful to add context about why this needs to be split and the reason for adding the retries here in a comment.